fix(docs): content review #5 (#2100)
scaleway/docs-content: tutorials/ruby-on-rails/index.mdx

## Installing and configuring of PostgreSQL

> Ruby on Rails application typically use a SQL database for data persistence.

Reviewer: ```suggestion
Ruby on Rails application typically uses an SQL database for data persistence.
```
